calour/io.py

```diff
@@ -432,131 +516,105 @@ def read_open_ms(data_file, sample_metadata_file=None, gnps_file=None, feature_m
 
     Returns
     -------
-    Experiment
+    MS1Experiment
 
     See Also
     --------
     read
     '''
-    logger.debug('Reading OpenMS data (OpenMS bucket table %s, map file %s)' % (data_file, sample_metadata_file))
-    if rows_are_samples:
-        data_file_type = 'openms_transpose'
+    if file_type == 'mzmine2':
+        if rows_are_samples is None:
+            rows_are_samples = False
+        if table_sep is None:
```

Comment:
you can set `','` instead of `None` as default

Reply:
i think we should keep none, since this way we can set the default based on the data_file_type while enabling the user to change if needed. if we set default as ',' we won't be able to change with other separators based on file type (say \t if needed)?